p2p/libp2p/netMessenger.go

```diff
@@ -347,60 +350,79 @@ func (netMes *networkMessenger) createConnectionsMetric() {
 	netMes.p2pHost.Network().Notify(netMes.connectionsMetric)
 }
 
-func (netMes *networkMessenger) printLogs() {
+func (netMes *networkMessenger) printAddresses() {
 	addresses := make([]interface{}, 0)
 	for i, address := range netMes.p2pHost.Addrs() {
 		addresses = append(addresses, fmt.Sprintf("addr%d", i))
 		addresses = append(addresses, address.String()+"/p2p/"+netMes.ID().Pretty())
 	}
 	log.Info("listening on addresses", addresses...)
-
-	go netMes.printLogsStats()
-	go netMes.checkExternalLoggers()
 }
 
-func (netMes *networkMessenger) printLogsStats() {
+func (netMes *networkMessenger) doAuxiliaryJobs() {
```

Comment:
rename to printAuxiliaryLogs ?

Reply:
struck my mind when I've wrote the function but it does something else but prints: it periodically checks if it has to switch some external libs loggers (calling `setupExternalP2PLoggers` function). I think it is better to keep it as it is.